### PR TITLE
Remove Lerna

### PR DESCRIPTION
**Pull request by ran-dall**
Remove Lerna, in favor of integrating everything into PNPM.

Signed-off-by: Randall <ran.dall@icloud.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,11 +7,10 @@
     "url": "https://github.com/withastro/compiler.git"
   },
   "scripts": {
-    "build": "make wasm && lerna run build --scope @astrojs/compiler",
-    "build:compiler": "lerna run build --scope @astrojs/compiler",
+    "build": "make wasm",
```
Comment (natemoo-re) [inline package.json:10]:
```suggestion
    "build": "make wasm",
    "build:compiler": "pnpm --filter @astrojs/compiler run build",
```

File: package.json [hunk 2]
```diff
@@ -7,11 +7,10 @@
...
     "lint": "eslint \"packages/**/*.{cjs,js,jsx,mjs,ts,tsx}\"",
     "format": "prettier -w .",
-    "prerelease": "pnpm run build:compiler",
+    "prerelease": "pnpm run build",
```
Comment (natemoo-re) [inline package.json:13]:
```suggestion
    "prerelease": "pnpm run build:compiler",
```

**Review by natemoo-re — COMMENTED** [review]
Nice! I think this still needs handling for the `@astrojs/compiler` package.

Also would love to remove the `lerna` dependency and update the lockfile.

File: package.json (hunk 1 above)
Comment (ran-dall) [reply package.json:10]: @natemoo-re You know, I looked for like an hour for that `--filter` command. LOL

**Review by natemoo-re — APPROVED** [review]
Thanks @ran-dall!

**Pull request merged**